Q2.txt

```diff
@@ -0,0 +1,18 @@
+javascript:
+var a = ["john", "mike", "susan", "matt", "joe", "larry", "ken", "samantha", "jenny", "Sara", "henry"];
+var x=a.length -1;
+while (x>0){
```

Comment:
it should be x >= 0 since 0 is a valid element position in the array, otherwise, x > 0 will skip 0